clang-tools-extra/clang-tidy/modernize/UseStdPrintCheck.cpp

```diff
@@ -20,6 +20,11 @@ namespace clang::tidy::modernize {
 
 namespace {
 AST_MATCHER(StringLiteral, isOrdinary) { return Node.isOrdinary(); }
+AST_MATCHER(QualType, isSimpleChar) {
+  const auto ActualType = Node.getTypePtr();
+  return ActualType->isSpecificBuiltinType(BuiltinType::Char_S) ||
+         ActualType->isSpecificBuiltinType(BuiltinType::Char_U);
+}
```

Comment:
consider moving this to some exist file in utils directory

Reply:
OK. I probably ought to do the same with `isOrdinary` too, but I'll do that in a separate change.